test/e2e/cmd/workspaces_test.go

```diff
@@ -36,48 +34,106 @@ import (
 const (
 	testResultsDirectory = "/tmp/artifacts"
 	jUnitOutputFilename  = "junit-workspaces-operator.xml"
+	testServiceAccount   = "terminal-test"
 )
 
+func waitDeletionWorkspace() error {
+	var isRemoved = false
+	var err error
+	thresholdAttempts := 10
+	delayBetweenAttempts := 1
+
+	for i := 0; i < thresholdAttempts; i++ {
+		isRemoved, err = config.AdminK8sClient.IsNamespaceExist(config.WorkspaceNamespace)
+		if isRemoved {
+			break
+		}
+		if err != nil || !isRemoved {
+			fmt.Println("The created namespace " + config.WorkspaceNamespace + " is not removed yet " + err.Error())
```

Comment:
Well, I changed every place to use log, but now I see that `ginkgo` uses fmt inside as well.
BTW I would keep logger since I like timestamp we get with it )
```
2020/12/15 20:12:31 Cleaning up test namespace test-terminal-namespace
2020/12/15 20:12:31 Namespace 'test-terminal-namespace' is in Terminating phase. Waiting until it's removed
2020/12/15 20:12:32 Namespace 'test-terminal-namespace' is in Terminating phase. Waiting until it's removed
2020/12/15 20:12:33 Namespace 'test-terminal-namespace' is in Terminating phase. Waiting until it's removed
2020/12/15 20:12:34 Namespace 'test-terminal-namespace' is in Terminating phase. Waiting until it's removed
2020/12/15 20:12:35 Namespace 'test-terminal-namespace' is in Terminating phase. Waiting until it's removed
2020/12/15 20:12:36 Namespace 'test-terminal-namespace' is in Terminating phase. Waiting until it's removed
2020/12/15 20:12:37 Namespace 'test-terminal-namespace' is in Terminating phase. Waiting until it's removed
2020/12/15 20:12:38 Namespace 'test-terminal-namespace' is in Terminating phase. Waiting until it's removed
2020/12/15 20:12:39 Namespace 'test-terminal-namespace' is in Terminating phase. Waiting until it's removed
```

Reply:
Done